src/components/editor/property/OutlineHeader.jsx

```diff
@@ -40,10 +42,41 @@ OutlineHeader.propTypes = {
   collapsed: PropTypes.any,
   handleCollapsed: PropTypes.func,
   id: PropTypes.string,
-  isRequired: PropTypes.any,
-  isAdd: PropTypes.bool,
-  pt: PropTypes.object,
-  spacer: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+  property: PropTypes.object,
+  reduxPath: PropTypes.array,
+  resourceModel: PropTypes.object,
+  handleToggleAddButton: PropTypes.func,
```

Comment:
Why is it `handleToggleAddButton` rather than `handleAddButton`?  Is it toggleable?

Reply:
It actually handles 2 buttons: the toggle (plus) and the add button (which is add and then toggle).

Reply:
To me toggle means open and close.  So, `handleAddAndOpen` is preferable in that case.  I think we could also call this `handleAdd` as that is the users intent.  The fact that "add" has two actions associated with it is the business of a the reducer.

Reply:
Renamed.